src/fragments.js

```diff
@@ -27,6 +27,7 @@ export const fragments = graphql`
         lng
       }
     }
+    alsoOffering
```

Comment:
Can we move this up so it's alphabetical (excepting `locations` which has a bunch of subfields)?